docs/specification/draft/basic/authorization.mdx

```diff
@@ -82,14 +82,18 @@ authorization servers to MCP clients, as well as the discovery process through w
 clients can determine authorization server endpoints and supported capabilities.
 
 ### 2.3.1 Authorization Server Location
+
 MCP servers **MUST** implement OAuth 2.0 Protected Resource Metadata ([RFC9728](https://datatracker.ietf.org/doc/html/rfc9728))
 specification to indicate the locations of authorization servers. The Protected Resource Metadata document returned by the MCP server **MUST** include
 the `authorization_servers` field containing at least one authorization server.
```

Comment:
Should we recommend that there is only one listed for simplicity sake?

Reply:
I think that makes sense. I will tweak the verbiage.